script/repackage-dylibs.rb

```diff
@@ -89,6 +89,7 @@ def repackage_dependency(dep)
 
     note "Copying #{dep[:path]} to TARGET_FRAMEWORKS_PATH"
     FileUtils.cp dep[:path], TARGET_FRAMEWORKS_PATH
+    FileUtils.chmod "u=wr", dep[:path]
```

Comment:
This is `chmod`ing the wrong file; `dep[:path]` is the library's path in Homebrew. `chmod`ing the copied library would be

```ruby
FileUtils.chmod "u=wr", File.join(TARGET_FRAMEWORKS_PATH, dep.name)
```